deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
As I said, I don't really mind the dlopen, but, since this library also contains the `unw_*` symbols that will cause quite a bit of mess in the code that I'm not sure is worth it. If you think it's worth it, feel free to go ahead and change it though.